ext/zend_test/tests/gh17899.phpt

```diff
@@ -0,0 +1,34 @@
+--TEST--
+GH-17899 (zend_test_compile_string with opcache crash on invalid script path)
+--EXTENSIONS--
+zend_test
+--INI--
+opcache.enable_cli=1
+--CREDITS--
+YuanchengJiang
+--FILE--
+<?php
+$source = '<?php
+require("sumfile.php");
+?>';
+try {zend_test_compile_string($source,$source,$c);} catch (Exception $e) { echo($e); }
```

Comment:
Please add closing `?>` tags to the FILE section.

Reply:
right..